examples/example3/example3.js

```diff
@@ -295,26 +295,28 @@ appExample3.controller(
                     var images = data.documentElement.getElementsByTagName('url');
                     // images is a psudo-array, not a real array. using slide to make it a real array.
                     images = Array.prototype.slice.call(images);
-                    // loop through all the url elements
-                    images.forEach(function(image) {
-                        // grab the url string and put it in imageQueue
-                        $scope.imageQueue.push(image.innerHTML);
+                    images.forEach(function (image) {
+                        if (image && image.childNodes[0] && image.childNodes[0].nodeValue) {
+                          $scope.imageQueue.push(image.childNodes[0].nodeValue);
```

Comment:
`images.innerHTML` was undefined, so arrays fill up with undefined instead of URLs, and no other operations work as expected without using `childNodes[0].nodeValue` here